src/devices/MAX31856/MAX31856.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.IO;
+using UnitsNet;
+
+namespace Iot.Device.MAX31856
```

Comment:
Per our [coding guidelines](https://github.com/dotnet/iot/blob/master/Documentation/Coding-guidelines.md#coding-guideliness) this should be spelled as `Iot.Device.Max31856`

Reply:
I updated all code to reflect Max31856 vice MAX31856